package.json

```diff
@@ -35,8 +35,11 @@
         "onLanguage:csharp",
         "onCommand:o.restart",
         "onCommand:o.pickProjectAndStart",
-        "onCommand:dotnet.restore",
+        "onCommand:o.restore",
+        "onCommand:o.execute",
         "onCommand:o.showOutput",
+        "onCommand:o.execute-last-command",
+        "onCommand:dotnet.restore",
```

Comment:
And note that I completely glossed over your recommendation to review this as two commits. Sorry about that.